Laravel-Backpack/CRUD
# [Feature Request] Allow delete button on edit page

## WHY

### BEFORE - What was wrong? What was happening before this PR?

Nothing, it is a feature requested [here](https://github.com/Laravel-Backpack/ideas/issues/208).

![delete-button-on-edit](https://user-images.githubusercontent.com/33960976/205296335-e075358f-90b0-48ad-a2a5-3f5a669abcbd.gif)

### AFTER - What is happening after this PR?

The new feature is added.


## HOW

### How did you achieve that, in technical terms?

Only one file was changed. A button is added in `src/resources/views/crud/inc/form_save_buttons.blade.php`.


### Is it a breaking change?

No.


### How can we test the before & after?

Make sure that the buttons shows up when using `$this->crud->setOperationSetting('showDeleteButton', true);`.

**Important:** documentation also is being updated [here](https://github.com/Laravel-Backpack/docs/pull/395).
